src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/DueDiligenceValidator.java

```diff
@@ -28,8 +28,13 @@ public DueDiligenceValidator(AddressDtoValidator addressDtoValidator) {
         this.addressDtoValidator = addressDtoValidator;
     }
 
-    public Errors validate(DueDiligenceDto dueDiligenceDto, Errors errors, String loggingContext) {
+    public Errors validateWithIdentityDate(DueDiligenceDto dueDiligenceDto, Errors errors, String loggingContext) {
```

Comment:
i think it makes better sense to keep 'validate' as the full 'everything' validator and have a 'validateWithoutIdentityDate' so switching the naming around but having 'without' instead of 'with'.

Reply:
Swapped names around